config/locales/en/claims/support/schools/claims.yml

```diff
@@ -28,8 +28,17 @@ en:
             grant_funding: Grant funding
             hours: hours
             page_caption: Claims - %{school_name}
+            remove_claim: Remove claim
           new:
             page_title: Accredited provider - Add claim
+          remove:
+            page_title: Accredited provider - Remove claim
```

Comment:
Is the page title here correct? I'm not sure about the `Acredited provider bit` I think it needs to be `Are you sure you want to remove this claim?` At least that's what's on the prototype https://itt-mentoring-beta-dev-a238297da236.herokuapp.com/support/organisations/049fc44e-6876-4884-8a77-d363173ce8d2/claims/undefined/delete